open_humans/forms.py

```diff
@@ -69,11 +73,19 @@ class MemberSignupForm(AccountSignupForm):
     class Meta:  # noqa: D101
         fields = '__all__'
 
+    def clean(self):
+        super().clean()
+
+        if 'terms' not in self.cleaned_data:
+            self.add_error('terms', _('You must accept our terms of service.'))
```

Comment:
(minor) we use "terms of use" instead of "terms of service"

but also... I'm not sure I see this phrase in the form when I try using it?

Reply:
Ah, oops, yeah, this is handled in js.